internal/worker/api/errors.go

```diff
@@ -148,18 +148,30 @@ func apiErrorFromEchoError(echoError *echo.HTTPError) ServiceErrorCode {
 
 // Convert an echo error into an AOC compliant one so we send a correct json error response
 func HTTPErrorHandler(echoError error, c echo.Context) {
-	doResponse := func(code ServiceErrorCode, c echo.Context, internal error) {
+	doResponse := func(code ServiceErrorCode, c echo.Context) {
 		if !c.Response().Committed {
 			var err error
 			sec := find(code)
 			apiErr := APIError(code, sec, c)
 
+			internalError, internalErrorOk := echoError.(*echo.HTTPError)
```

Comment:
Hm now there's a double cast here, cloudapi does it too and I think it's much cleaner if `doResponse` just gets the internal error as an argument in case it needs to log.